### PR TITLE
Adding method for String.valueOf() method

### PR DESCRIPTION
**Pull request by irenesmith**

### CODE REVIEW

File: live-examples/js-examples/string/meta.json [hunk 1]
```diff
@@ -20,6 +20,13 @@
             "fileName": "string-trimstart.html",
             "title": "JavaScript Demo: String.trimStart()",
             "type": "js"
+        },
+        "stringvalueOf": {
```
Comment (schalkneethling) [inline live-examples/js-examples/string/meta.json]: Super Nit: I would probably change this to `"stringValueOf"`

File: live-examples/js-examples/string/string-valueOf.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<pre>
+<code id="static-js">const theAnswer = '42';
+console.log(theAnswer.valueOf()); // 42
```
Comment (schalkneethling) [inline live-examples/js-examples/string/string-valueOf.html]: We use the following format:

```
console.log(theAnswer.valueOf());
// expected output: 42
```

**Comment by wbamberg** [comment]
@Elchi3 , I just updated this example along the lines of [your comment](https://github.com/mdn/interactive-examples/pull/1076#issuecomment-413519980), could you please review?

**Comment by schalkneethling** [comment]
Thanks @Elchi3 ☕️

**Pull request merged**